### PR TITLE
Feature/new qc spreadsheet

### PR DESCRIPTION
**Pull request by ericearl**
@GregConan latest edits for the abcd_fastqc01 spreadsheet integration

### CODE REVIEW

File: spreadsheets/README.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 This is where the spreadsheets belong:
 
-1. `DAL_ABCD_merged_pcqcinfo.csv` (the DAIC QC info)
-1. `image03.txt` (the NDA DICOM imaging data info)
+1. `abcd_fastqc01.txt` (the QC info)
 1. `ABCD_good_bad_series_table.csv` (generated after `data_gatherer` is run)
 
-The `DAL_ABCD_merged_pcqcinfo.csv` is currently only available from the ABCD DAIC. So, it is only accessible for someone within the ABCD consortium (who can access ABCD data on the NDA website). However, the MRI QC scores will soon be uploaded to NDA. Afterwards, this script will be updated to download that file and read from it instead of requiring the user to manually get the `DAL_ABCD_merged_pcqcinfo.csv` spreadsheet.
+The `abcd_fastqc01.txt` is currently available from the NDA. In a future update, this script will be updated to download that file and read from it instead of requiring the user to manually get the `abcd_fastqc01.txt` spreadsheet.
```
Comment (ericearl) [inline spreadsheets/README.md]: Is that even possible?  I feel like we shouldn't promise something we are not sure is possible.

File: src/sefm_eval_and_json_editor.py [hunk 2]
```diff
@@ -267,10 +274,15 @@ def main(argv=sys.argv):
     # for this script's usage of FSL_DIR...
     fsl_dir = args.fsl_dir + '/bin'
 
+    # This block was added by Greg Conan 2019-10-25
+    for json_file in os.scandir(args.output_dir):
+        json_path = json_file.path
+        if "json" in json_path:
```
Comment (ericearl) [inline src/sefm_eval_and_json_editor.py]: should you look maybe for explicitly an ending extension of `.json`?

File: src/sefm_eval_and_json_editor.py [hunk 3]
```diff
@@ -297,24 +309,30 @@ def main(argv=sys.argv):
...
             sefm_metadata = layout.get_metadata(sefm)
+
+            """
             if 'Philips' in sefm_metadata['Manufacturer']:
                 insert_edit_json(sefm_json, 'EffectiveEchoSpacing', 0.00062771)
             if 'GE' in sefm_metadata['Manufacturer']:
                 insert_edit_json(sefm_json, 'EffectiveEchoSpacing', 0.000536)
             if 'Siemens' in sefm_metadata['Manufacturer']:
                 insert_edit_json(sefm_json, 'EffectiveEchoSpacing', 0.00051001152626)
+            """
```
Comment (ericearl) [inline src/sefm_eval_and_json_editor.py]: Are these being commented out?  That would be wrong.

File: src/sefm_eval_and_json_editor.py [hunk 4]
```diff
@@ -297,24 +309,30 @@ def main(argv=sys.argv):
...
                 #if 'T1' in TX_metadata['SeriesDescription']:
+
+            """
             if 'Philips' in TX_metadata['Manufacturer']:
                 insert_edit_json(TX_json, 'DwellTime', 0.00062771)
             if 'GE' in TX_metadata['Manufacturer']:
                 insert_edit_json(TX_json, 'DwellTime', 0.000536)
             if 'Siemens' in TX_metadata['Manufacturer']:
                 insert_edit_json(TX_json, 'DwellTime', 0.00051001152626)
+            """
```
Comment (ericearl) [inline src/sefm_eval_and_json_editor.py]: Are these being commented out?  That would be wrong.

File: abcd2bids.py [hunk 5]
```diff
@@ -4,46 +4,52 @@
...
 import signal
 import subprocess
 import sys
 
 # Constant: List of function names of steps 1-5 in the list above
 STEP_NAMES = ["create_good_and_bad_series_table", "download_nda_data",
-              "unpack_and_setup", "correct_jsons", "run_bids_validator"]
+              "unpack_and_setup", "correct_jsons", "validate_bids"]
+
+PWD = os.getcwd()
```
Comment (ericearl) [inline abcd2bids.py]: Trusting the present working directory seems  a little risky.  Is there another way to do that?

File: src/sefm_eval_and_json_editor.py (hunk 3 above)
Comment (GregConan) [reply src/sefm_eval_and_json_editor.py]: The problem there is that the Manufacturer field is not included in the abcd_fastqc01 spreadsheet. To get the Manufacturer field, it must be either (1) read by merging abcd_fastqc01 with image03, or (2) extracted from each .tgz file after it has been downloaded and retroactively added to the good_and_bad_series spreadsheet. I was working on (2) before someone said that getting the Manufacturer data may not be needed.

Comment (perronea) [reply src/sefm_eval_and_json_editor.py]: At this point the Manufacturer field is in the metadata. It's extracted from the dicom itself during dcm2bids and put in the sidecar json. image03 is not needed, but this code block is.

**Comment by ericearl** [comment]
@GregConan , I think @perronea and I have done our review.  Make these corrections ASAP and test them out again on a small subset of subjects.

**Comment by ericearl** [comment]
@perronea I like the look of it after @GregConan 's commit yesterday.  If you approve, I'll merge it.

**Comment by perronea** [comment]
Looks good. As long as he's done some basic checks on the results of using the fastqc spreadsheet (make sure that the number of valid subjects returned when using the fastqc spreadsheet matches the number used with the previous method) then push it.

**Comment by ericearl** [comment]
@perronea Is there any way to do that without converting the *entire* dataset again?

**Comment by perronea** [comment]
There should be a logging function where you can choose not to actually download all the images, but instead just produce the commands/summarize what could be downloaded

**Comment by ericearl** [comment]
@perronea Great idea!  However, I'm not considering that a necessary feature of this pull request or this release.  @GregConan please summarize Anders' idea into a GitHub issue here and I am merging this pull request to be released as v1.0.0.

**Comment by ericearl** [comment]
Oops, I meant v2.0.0.

**Pull request merged**